episodes/chap6_setup_node.md

```diff
@@ -251,9 +246,7 @@ then it will start the node and print out the following messages:
 [warn ] - private key file provided ***/private_key.pem, but does not exists
 [info ] - Setting up databases
 [info ] -   Processing csv database default: ***/data.csv
-[debug] -   - non file-based database added
```

Comment:
I think that, if the db file has been correctly mounted, the log says "file-based database added". Should we leave the correct log line?

Reply:
It's debug message, safe to remove it for end users.